src/sentry/api/serializers/models/group.py

```diff
@@ -392,6 +395,16 @@ def _collapse(self, key) -> bool:
             return False
         return key in self.collapse
 
+    def _get_priority(self, obj: Group) -> str:
+        priority = obj.priority
+        if priority == PriorityLevel.HIGH:
+            priority_label = "high"
+        elif priority == PriorityLevel.MEDIUM:
+            priority_label = "medium"
+        else:
```

Comment:
Should `PriorityLevel.LOW` be a default? Do we need to worry about a case where Group Priority is None?

Reply:
The default priority should be `PriorityLevel.MEDIUM`, per https://www.notion.so/sentry/Tech-Spec-Priority-in-Issue-Details-Stream-ac9ecbbc5cfd4baaae39ecf32f8e74ce?pvs=4#ae9b3ffebd4a4859b940f5b50e2ddba3

Reply:
We should add a feature flag check here, and support none until the migration is complete and we're actively setting priority on each new issue

Reply:
Ah yes, thanks for the reminder!